### ISSUE TITLE
It dosn't work well when I use vue2.0( I develop the server end with nodejs)

### ISSUE THREAD

**Issue by Zenfeder** [MetinSeylan/Vue-Socket.io#10]
SOS!!! The server-end can receive the event emitted by client-end, but the client-end can't receive the event from server-end. I show my code as below:

```javascript
// client.vue (client-end)
<template>
   ...
</template>

<script>
export default {
    sockets: {
        connect: function () {
            console.log('socket connected');  //  This cmd dosn't works
        },
        serverEmit: function (msg) {
            console.log("Message："+msg);  //  This cmd dosn't works
        }
    },
    methods: {
        trigger () {
            this.$socket.emit('clientEmit', "I'm a message from client."); //  This cmd works well
        }
    }
}
</script>
```
```javascript
// server.js(server-end)
io.on('connection', (socket) => {
        console.log('Socket connect successful.'); //  This cmd works well

        socket.on('clientEmit', (msg) => {
            console.log("Message: " + msg);

            socket.emit('serverEmit', "I'm a message from server."); //  This cmd works well
        });
    });
```

I think the option  'sockets' is not useful in vue2.0 anymore.

**Comment by MetinSeylan** [comment]
vue 2.0 support not yet :(

**Comment by MetinSeylan** [comment]
published 2.0.0 https://github.com/MetinSeylan/Vue-Socket.io/commit/7f5fac27d0623ccadd36af3f6b5fcf72eb5c6445
